client/html/layouts/basket/product/body-default.html

```diff
@@ -0,0 +1,114 @@
+<?php
```

Comment:
No main-header-default.html and main-body-default.html stays the same because they belong to the "Main" subpart